modules/ossm-control-plane-deploy.adoc

```diff
@@ -3,37 +3,75 @@
 // * service_mesh/service_mesh_install/installing-ossm.adoc
 
 [id="ossm-control-plane-deploy_{context}"]
+= Deploying the {ProductName} control plane
 
-= Deploying the control plane
+You can deploy the {ProductShortName} control plane by using the {product-title} web console or the CLI.
 
-With the introduction of {product-title} 4.1, the network capabilities of the host are now based on nftables rather than iptables. This change impacts the initialization of the {ProductName} application components. {ProductShortName} needs to know what host operating system OpenShift is running on to correctly initialize {ProductShortName} networking components.
+[id="ossm-control-plane-deploy-operatorhub_{context}"]
+= Deploying the control plane with the web console
 
+Follow this procedure to deploy the {ProductName} control plane by using the web console.
+
+.Prerequisites
+
+* The {ProductName} Operator must be installed.
+* Review the xref:../service_mesh_install/customizing-installation-ossm.adoc#customize-installation-ossm[Customize the {ProductName} installation] instructions.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Create a new project named `istio-system`.
+
+. Navigate to *Catalogs* -> *Installed Operators*.
+
+. Click the {ProductName} Operator.
+
+. Under *Provided APIs*, the Operator enables you to create two resource types:
+** A `ServiceMeshControlPlane` resource
+** A `ServiceMeshMemberRoll` resource
+
+. Click *Create New* under *Istio Service Mesh Control Plane*.
+
+. Modify the minimal `ServiceMeshControlPlane` template.
```

Comment:
@knrc @geekspertise 
Based on the default SMCP yaml provided by the operator, we may need to add an important note about altering the jaeger template from the production default to the all-in-one as the production template requires a great deal more resources and will likely cause the service mesh install to not complete successfully.

Reply:
Note the default within the metadata and operator image is now to use all-in-one instead of production-elasticsearch.

Reply:
Removing this based on the change to the metadata in the operator changing to default to the all-in-one template.